ktor-server/ktor-server-plugins/ktor-server-cors/common/src/io/ktor/server/plugins/cors/routing/CORS.kt

```diff
@@ -24,5 +25,9 @@ import io.ktor.server.plugins.cors.*
  * [Report a problem](https://ktor.io/feedback/?fqname=io.ktor.server.plugins.cors.routing.CORS)
  */
 public val CORS: RouteScopedPlugin<CORSConfig> = createRouteScopedPlugin("CORS", ::CORSConfig) {
+    route?.options("{cors-options-wildcard...}") {
```

Comment:
Could you tell me why we need an explicit handler?

Reply:
Otherwise, this route will not be resolved, and thus, the `onCall` handler of the CORS plugin will not be called.